src/platforms/android/common/performance/included-instrumentation.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Included Instrumentation
+sidebar_order: 21
+description: "Learn what transactions are captured after tracing is enabled."
+---
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+<Note>
+
+This is only available since Sentry's Android SDK version `4.3.0`
+
+</Note>
+
+### Android's Activity Instrumentation
+
+The Activity's instrumentation, once enabled, captures transactions for each Activity's start up, Transaction names follow pattern `ActivityName`, for example for an Activity called `MainActivity`, The Transaction's name is `MainActivity` and its operation is `navigation`.
+
+The transaction starts before each Activity's `onCreate` method is called.
```

Comment:
`m`: Maybe we should mention that we hook into `onActivityPreCreated`.

Reply:
not sure, that's impl. details, and with an user friendly desc. `before each Activity's `onCreate` method is called` which is what `onActivityPreCreated` does